Package.swift

```diff
@@ -3,6 +3,27 @@
 
 import PackageDescription
 import class Foundation.ProcessInfo
+import struct Foundation.URL
+
+// Extract compiler flags from Local.xcconfig, if any.
+var additionalCompilerFlags: [PackageDescription.SwiftSetting] = {
+    guard let config = try? String(
+        contentsOf: URL(fileURLWithPath: #filePath)
+            .deletingLastPathComponent()
+            .appendingPathComponent("Local.xcconfig")
+    ) else {
+        return []
+    }
+    // We split the capture group by space and remove any special flags, such as $(inherited).
+    return config
+        .firstMatch(of: #/^SWIFT_ACTIVE_COMPILATION_CONDITIONS *= *(.*)$/#.anchorsMatchLineEndings())?
+        .output
+        .1
+        .split(whereSeparator: \.isWhitespace)
+        .filter { !$0.isEmpty && !$0.hasPrefix("$") }
+        .map { .define(String($0)) }
+        ?? []
+}()
```

Comment:
This seems like a great idea, but it also means that if you have a `Local.xcconfig` defined for your app, we're also going to read those, right? 
Maybe in turn, we should make sure that we don't have any risk of name conflicts? Like, either by updating the `Local.xcconfig` file name or the flags themselves with an RC prefix / suffix. 

Like: `RevenueCatLocal.xcconfig` or `REVENUECAT_PAYWALL_COMPONENTS`

Reply:
Update: just read the other conversation right after asking about the same thing 😂 
Maybe we should document in a comment that it only applies if you have it defined in the exact folder, so that people using the library and looking at the code don't wonder about the same thing?

Reply:
That's a good idea to clear any doubts! How about: https://github.com/RevenueCat/purchases-ios/pull/4368/commits/3f57bafae9cdf1718444e9784acb7e57067bb008.

Reply:
:chefskiss: